frontend/public/locales/en/public.json

```diff
@@ -13,5 +13,6 @@
   "Pause": "Pause",
   "Warning": "Warning",
   "Not available": "Not available",
-  "Resource": "Resource"
+  "Resource": "Resource",
+  "No Pipelines Found": "No Pipelines Found"
```

Comment:
This should be in the Pipeline package not public

Reply:
Done.